src/db_api_util.ts

```diff
@@ -563,7 +565,14 @@ export async function archiveDemoVoter(
         AND twilio_phone_number = $2`,
       [userId, twilioPhoneNumber]
     );
-
+    await client.query(
+      `UPDATE threads
+      SET archived = true
+      WHERE
+        is_demo = true
+        AND user_id = $1`,
+      [userId]
```

Comment:
can you use twilioPhoneNumber here, to be consistent with the three other queries above? if threads doesn't have twilio_phone_number then it's fine -- we can fix this after the election.